en_US/deploy/cluster/create-cluster.md

```diff
@@ -1,140 +1,142 @@
-# Create and manage clusters
+# Create an EMQX Cluster
 
-EMQX supports creating clusters manually and automatically. This chapter will guide you through creating and managing EMQX clusters.
+EMQX supports creating clusters manually and automatically. There are 2 key concepts before creating a cluster, node names and node discovery. 
 
-:::tip Prerequisites:
+## Concepts
 
-- Knowledge of [Distributed Clusters](./introduction.md).
-- Knowledge of [Deployment Architecture and Cluster Requirements](./mria-introduction.md).
-  :::
+### Node Names
 
-## Preparations before clustering
+Before starting the cluster creation step, let's first get familiar with the concept of node names in EMQX. EMQX nodes are identified by their names. A node name consists of two parts, node name and host, separated with `@`, for example, `emqx@s1.emqx.io`. The host part must either be the IP address or FQDN (`myhost.example.domain`), for example:
 
-1. All nodes are set with a unique node name in the format of `name@host`, where host must be an IP address or fully qualified domain name (FQDN).
-2. If there is a firewall or security group between nodes, ensure the cluster communication port has been opened. For details, see [Intra-cluster communication port](./security.md).
-3. All nodes use the same cookie.
-
-:::tip
+- For EMQX node deployed on server `s1.emqx.io`, the node name should be `emqx@s1.emqx.io`; 
+- If this server has a static IP (`192.168.0.10`), the node name should be `emqx@192.168.0.10`. 
 
-EMQX node names are immutable, as they are baked into the database schema and data files. It is strongly recommended to use static FQDNs for EMQX node names, even when the nodes have static IPs.
+::: tip Tip
+EMQX node names are immutable, as they are baked into the database schema and data files. Therefore, it is recommended to use static FQDNs for EMQX node names.
```

Comment:
I think the part about preferring FQDNs to static IPs is important, since we had incidents related to this.